app/views/layouts/application.html.erb

```diff
@@ -313,7 +313,9 @@
   <% end %>
   <div id='aria_alerts' class='hide-text affix' role="alert" aria-live="assertive"></div>
   <%= render :partial => 'layouts/foot', :locals => { :include_common_bundle => true } %>
-
+  <div id="bz-help-button">
+    <a target="_BLANK" href="https://help.bebraven.org/questions/ask/">Help</a>
```

Comment:
Ideally, this would be configurable but since we don't currently have a staging and prod instance (and I can't see us changing the domain anytime soon), I'm fine with it hardcoded.